typebinder_web_playground/index.html

```diff
@@ -5,17 +5,23 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/assets/bulmaswatch-default.min.css">
+    <script src="https://kit.fontawesome.com/6f34660bff.js" crossorigin="anonymous"></script>
```

Comment:
Is it possible (license-wise, mostly) to download this instead of linking to it ? 

I dont like that this might break if the URL change, and the potential privacy issues like we have with CDNs.

Reply:
Bulmawatch is MIT licensed and FontAwesome, we do not actually need. I am going to remove it.